src/controllers.php

```diff
@@ -163,6 +164,10 @@
         $link->setAttribute('class', 'anchor');
         $node->appendChild($link);
 
+        if (!isset($first_title) && strlen($title) > 0) {
```

Comment:
you should use the variable name ``$firstTitle``, and this condition can then become ``if (null === $firstTitle)``